Makefile

```
@@ -8,7 +8,7 @@ stable:
 	docker build -f Dockerfile --build-arg VERSION_OCTOPUS=${VERSION_OCTOPUS} -t octopus .
 
 develop:
-	docker build -f Dockerfile --build-arg VERSION_OCTOPUS=develop -t octopus-develop .
+	docker build -f Dockerfile --build-arg VERSION_OCTOPUS=main -t octopus-develop .
```

```suggestion
	docker build -f Dockerfile --build-arg VERSION_OCTOPUS=main -t octopus-main .
```
Be consistent, and then also name the docker image `-main` if we make the transition from `develop` to `main`?

---

I left this intentionally `develop` here as I wanted it to reflect the fact that it is a "developmental" version of octopus, and that its not the "main" version of octopus.
I can see why you would like to keep the consistency. Lets maybe discuss this tomorrow

---

I am now thinking of implementing the ability to specify any branch as well. Based on #31.
so the `octopus-develop` container would host any development versions ( main or any other branch, recognised by alpha-numeric version argument). and the `octopus` image would host stable releases ( characterised by numeric version argument)

---

Okay - let's put this on hold for now then.